R/pkg/R/DataFrame.R

```diff
@@ -3745,3 +3745,26 @@ setMethod("hint",
             jdf <- callJMethod(x@sdf, "hint", name, parameters)
             dataFrame(jdf)
           })
+
+#' alias
+#'
+#' @aliases alias,SparkDataFrame-method
+#' @family SparkDataFrame functions
+#' @rdname alias
+#' @name alias
+#' @examples
```

Comment:
Done, but do we actually need this? We don't use roxygen to maintain `NAMESPACE`, and (I believe i mentioned this before) we `@export` objects which are not really exported. Just saying...